.eslintrc.js

```diff
@@ -1,17 +1,31 @@
 module.exports = {
   env: {
-    browser: true,
     es2021: true,
-    node: true,
-    mocha: true,
+    'shared-node-browser': true,
   },
-  extends: ['airbnb-base', 'prettier', 'plugin:import/typescript'],
+  extends: [
+    'airbnb-base',
+    'prettier',
+    'plugin:import/typescript',
+    'eslint:recommended',
+  ],
```

Comment:
These changes aren't explicitly necessary, but I thought I would include them since I'm making changes to our eslint settings.
* `shared-node-browser` causes eslint to only recognize global variables present in _both_ node and browser contexts, instead of both.
* I removed `mocha` from the global list of environments in favor of adding `/* eslint-env mocha */` at the top of each mocha test file. This was we won't accidentally reference mocha global variables from our production files.
* `eslint:recommended` contains a set of recommended rules. It only caused one new issue in our codebase (which I fixed by configuring `no-constant-condition` below), so I figured it was a good idea to include.